eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -181,22 +181,21 @@ foreach ($service in $serviceNameList) {
   $clientPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('client' -eq $_.Type) })
   $clientPackages = $clientPackages | Sort-Object -Property Package
   foreach ($clientPackage in $clientPackages) {
-    $packageItems += GetClientPackageNode -clientPackage $clientPackage
+    $packageItems += GetPackageNode -package $clientPackage
   }
 
   # All management packages go under a single `Management` header in the ToC
   $mgmtPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('mgmt' -eq $_.Type) })
   $mgmtPackages = $mgmtPackages | Sort-Object -Property Package
-  if ($mgmtPackages) {
-    $children = &$GetDocsMsTocChildrenForManagementPackagesFn `
-      -packageMetadata $mgmtPackages `
-      -docRepoLocation $DocRepoLocation
-
+  $serviceReadmeBaseName = $service.ToLower().Replace(' ', '-').Replace('/', '-')
```

Comment:
Recommend keeping the declaration of `$serviceReadmeBaseName` closer to where it's used.